jerry-core/jcontext/jcontext.h

```diff
@@ -199,6 +211,11 @@ struct jmem_heap_t
  */
 extern jerry_context_t jerry_global_context;
 
+/*
+* If EXTERNAL_CONTEXT is not enable then JERRY_DEFINE_CURRENT_CONTEXT is a empty macro to avoid warnings
+*/
```

Comment:
Also out of style: not a doc comment (doesn't start with 2 asterisks) and asterisks are not indented by one space.

Typos: enable -> enabled, a -> an